ui/package.json

```diff
@@ -32,8 +32,10 @@
         "react-chartjs-2": "^2.11.2",
         "react-datepicker": "^4.25.0",
         "react-dom": "^16.14.0",
+        "react-markdown": "^8.0.7",
         "react-monaco-editor": "^0.50.1",
         "react-router-dom": "^4.2.2",
+        "remark-gfm": "^3.0.0",
```

Comment:
[On Slack](https://cloud-native.slack.com/archives/C0510EUH90V/p1704308933925729?thread_ts=1703894735.423229&cid=C0510EUH90V) we discussed potentially putting the whole markdown feature behind a feature flag as otherwise the deps load unconditionally for all users, even those who don't use this feature

Reply:
So we can do an initial code-split and only load the deps if the `title` and `description` annotations exist, but it may be difficult to discern if those have markdown specifically without a dep, which is where a feature flag might be good. I still would prefer not to gate it behind a feature flag, but I also don't want the deps loaded for users who don't use the feature.

Reply:
Code-split this in #12580 based on if the annotations exist, but as written above, can't quite determine if those have markdown without a dep, so there is potentially further action to do here. Maybe there's a clever one-line regex to do it though and then we wouldn't need a dep to detect markdown